lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -134,6 +147,19 @@ export class AccountEntity {
             account.authorityType = CacheAccountType.GENERIC_ACCOUNT_TYPE;
         }
 
+        let clientInfo: ClientInfo | undefined;
+
+        try {
+            if (accountDetails.clientInfo && cryptoObj) {
+                clientInfo = buildClientInfo(
+                    accountDetails.clientInfo,
+                    cryptoObj
+                );
+            }
+        } catch (e) {
+            throw e;
+        }
```

Comment:
Point of wrapping in a try/catch if all you're doing is rethrowing?

Reply:
I thought I might have to deal with the error somehow but not the case, removing try/catch